codestyle-config/pom.xml

```diff
@@ -22,16 +22,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
   <parent>
-    <groupId>org.apache.apex</groupId>
```

Comment:
I don't see why two different issues should be in a single PR. I would suggest to wrap up the checkstyle change and then address parent pom properly in a separate JIRA.